.circleci/config.yml

```
@@ -107,3 +108,19 @@ jobs:
           name: Run system tests
           command: |
             COVERAGE=false bundle exec rspec spec/system/
+
+
+workflows:
+  test-build-and-publish-image:
+    jobs:
+      - run-tests
+      - wait-for-approval:
+          type: approval
+      - docker/publish:
+          cache_from: $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME:latest
+          image: $CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME
```

Dockerhub doesn't allow `-` in usernames, so [I signed up as the user `stringerrss`](https://hub.docker.com/repository/docker/stringerrss/stringer). I could set it as an env variable on CircleCI, or we could hard code it here:

```suggestion
          cache_from: stringerrss/stringer:latest
          image: stringerrss/stringer
```

---

The example docs had these env vars within - I thought it was one of those things that it automatically provides certain env vars. I can hardcode it in :)

---

Yeah, I think CircleCI sets those env variables based on the Github org/repo, so we'd need to do something else because of the `-` in `stringer-rss`.